docs/installation/clients.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: documentation
+title: Client Installation
+sidebar: sidebars/documentation.html
+script: osmenu.js
+---
+
+# Client Installation
+
+Client machines can be added in one of two ways: manually or via a provisioned host.
+
+## Manual
+
+Install the appropriate Katello client release packages:
+
+<p>
+  Select your Operating System:
+  <select id="operatingSystems">
+     <option value="el5">Enterprise Linux 5 (CentOS, etc.)</option>
+     <option value="el6">Enterprise Linux 6 (CentOS, etc.)</option>
+     <option value="el7">Enterprise Linux 7 (CentOS, etc.)</option>
+     <option value="fc20">Fedora 20</option>
+     <option value="fc21">Fedora 21</option>
+  </select>
+</p>
+
+<div id="el5" markdown="1">
+```bash
+yum update -y http://fedorapeople.org/groups/katello/releases/yum/{{ site.version }}/katello/RHEL/5Server/x86_64/katello-client-repos-latest.rpm
```

Comment:
Small nitpick - can you link to client instead of Katello? e.g. https://fedorapeople.org/groups/katello/releases/yum/2.2/client/RHEL/6Server/x86_64/katello-client-repos-2.2.0-4.el6.noarch.rpm

That way, they can copy and paste that link w/out the package to sync the repository in Katello.